Makefile

```diff
@@ -0,0 +1,11 @@
+# If the first argument is "run"...
```

Comment:
@Kalachevskyi could you please help a bit with Makefile? Maybe you have some good examples and docs to read?